### PR TITLE
Show unavailabilities in event page

### PR DESCRIPTION
**Pull request by MarioRodrigues10**
Make sure when you're testing to also run https://github.com/coderdojobraga/bokken/pull/175

### CODE REVIEW

**Review by ruioliveira02 — COMMENTED** [review]
Remove the changes to `package-lock.json`

**Comment by MarioRodrigues10** [comment]
Link Checker is failing in `packages/ui/components/landing/Footer/config.tsx` although i didn't even touch that file.

File: apps/app/pages/admin/event/[id].tsx [hunk 1]
```diff
@@ -17,14 +17,12 @@ function EventPage() {
...
   const [enrolledNinjas, setEnrolledNinjas] = useState([]);
   const [availableMentors, setAvailableMentors] = useState<any[]>([]);
   const [unavailableMentors, setUnavailableMentors] = useState<any[]>([]);
   const [available, setAvailable] = useState<boolean>(true);
   useEffect(() => {
-    getAvailableMentors(event_id as string)
+    getMentors(event_id as string)
       .then((response: any) => {
         setAvailableMentors(
           response.data.filter((mentor: any) => mentor.is_available)
```
Comment (RuiL1904) [inline apps/app/pages/admin/event/[id].tsx:28]: This filtering would make the page not capable of showing the unavailable mentors.

File: apps/app/pages/admin/event/[id].tsx [hunk 2]
```diff
@@ -57,27 +61,65 @@ function EventPage() {
...
+                      title={`${mentor.first_name} ${mentor.last_name}`}
+                      description={mentor.notes ? `Notas: ${mentor.notes}` : ""}
+                    />
+                  </Link>
+                </List.Item>
+              )}
+            />
+          </>
+        )}
+      </>
```
Comment (RuiL1904) [inline apps/app/pages/admin/event/[id].tsx:122]: I'm sure you can use some inline conditions to not make this code this duplicated 😅

Comment (nelsonmestevao) [reply apps/app/pages/admin/event/[id].tsx:122]: It should be extracted to a component :bulb:

**Comment by RuiL1904** [comment]
@MarioRodrigues10 Could you please merge `main` into this branch? 🙏🏼
#183 fixes the CI failing.

**Review by RuiL1904 — COMMENTED** [review]
Make use of this component: https://ant.design/components/switch

**Review by ruioliveira02 — APPROVED** [review]

**Pull request merged**